README.md

```diff
@@ -116,4 +118,27 @@ docker run -d \
 
 ### Contribute to the project
 
-All contribs are welcome! Read the [CONTRIBUTING](CONTRIBUTING.md) documentation to get more information.
\ No newline at end of file
+All contribs are welcome! Read the [CONTRIBUTING](CONTRIBUTING.md) documentation to get more information.
+
+### TL;DR
+
+I want to monitor a host :
+(If kibana can't join elastic, check its network configuration.)
+
+```
+$ docker network create dockernet
+
+$ docker run -d --net=dockernet --name=elastic \
```

Comment:
@HipsterWhale Are you sure you want to write the TL;DR with `--net` instead of `--link` strategy?